app/seeders/basic_data/role_seeder.rb

```diff
@@ -26,176 +26,83 @@
 # See COPYRIGHT and LICENSE files for more details.
 #++
 module BasicData
-  class RoleSeeder < Seeder
-    def seed_data!
-      Role.transaction do
-        roles.each do |attributes|
-          Role.create!(attributes)
-        end
-
-        global_roles.each do |attributes|
-          GlobalRole.create!(attributes)
-        end
+  class RoleSeeder < ModelSeeder
+    self.model_class = Role
+    self.seed_data_model_key = 'roles'
```

Comment:
You are right. Yet I prefer the explicit approach here, even if it can feel a bit repetitive. 

I'm always a bit hesitant about this kind of topics, as deducing names by convention feels more "the Rails way", but showing a bit of the inners helps about understanding how the class works.